webspark.install

```diff
@@ -277,3 +277,24 @@ function webspark_update_9012(&$sandbox) {
   // Lock the configuration storage.
   \Drupal::state()->set('configuration_locked', TRUE);
 }
+
+/**
+ * Install imagemagick module and dependencies.
+ */
+function webspark_update_9013(&$sandbox) {
+  $module_installer = \Drupal::service("module_installer");
+
+  $module_installer->install(["file_mdm"]);
+  $module_installer->install(["sophron"]);
+  $module_installer->install(["imagemagick"]);
+
+  // Importing configs for ImageMagick module.
+  \Drupal::state()->set('configuration_locked', FALSE);
+  \Drupal::service('webspark.config_manager')->importConfigFile('file_mdm.file_metadata_plugin.getimagesize.yml');
+  \Drupal::service('webspark.config_manager')->importConfigFile('file_mdm.settings.yml');
+  \Drupal::service('webspark.config_manager')->importConfigFile('imagemagick.file_metadata_plugin.imagemagick_identify.yml');
+  \Drupal::service('webspark.config_manager')->importConfigFile('imagemagick.settings.yml');
+  \Drupal::service('webspark.config_manager')->importConfigFile('sophron.settings.yml');
+  \Drupal::service('webspark.config_manager')->importConfigFile('system.image.yml');
+  \Drupal::state()->set('configuration_locked', TRUE);
```

Comment:
@juanmitriatti The `importConfigFile` function requires that the file name not have the extension at the end. Also, there is a problem with that function. It should reference $filename, but it only references the undefined $name variable. So, it doesn't work. I have created a ticket that fixes that problem and issued a pull request. So, now this PR depends on that one (https://github.com/ASUWebPlatforms/webspark-module-webspark_utility/pull/23).

Reply:
Hi @tbutterf 
Pr has been updated, please check.
Thanks for your feedback.